cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -892,3 +892,59 @@ def _revert():
         if appliance.version < "5.11"
         else "Finished"
     )
+
+
+@pytest.mark.meta(automates=[1444092, 1515561])
+@pytest.mark.parametrize(
+    "credential", CREDENTIALS, ids=[cred[0] for cred in CREDENTIALS]
+)
+@pytest.mark.provider(
+    [RHEVMProvider, EC2Provider, VMwareProvider, AzureProvider], selector=ONE_PER_TYPE
```

Comment:
@sshveta I am uncollecting other providers because of Playbooks. Once I will add playbooks for other Providers like OSP, I will Add Marker to collect all Providers.

Reply:
Please add in-line comment/TODO for why specific providers are collected.

Reply:
I will add TODO/Comment in the #9076 after merge of it.

Reply:
Added Comment/TODO for other providers.